logging/access_test.go

```diff
@@ -293,3 +323,12 @@ func TestAccessLogStripQuery(t *testing.T) {
 	entry.Request.RequestURI += "?foo=bar"
 	testAccessLog(t, entry, logOutput, Options{AccessLogStripQuery: true})
 }
+
+func TestHashQueryParamValue(t *testing.T) {
+	want := uint64(3728699739546630719)
+	got := hash("foo")
+
+	if got != want {
```

Comment:
why not using require here also?

Reply:
I think these tests were created before testify dependency was added to skipper.
This test makes no sense. It just tests the library function, i'll remove.

Reply:
it's nice for coverage